### ISSUE TITLE
Penjelasan mengenai tiap analytic

### ISSUE THREAD

**Issue by gregoriusjimmy** [nodefluxio/inlabs#8]
Kita belum mempunyai demo untuk setiap analytic yang ada dan tidak semua developer mempunyai latar belakang computer vision dan. Contoh kasusnya, agar face recognition dapat menampilkan kandidat kita perlu melakukan enroll face menggunakan face enrollment dulu dan ini tidak ada di dokumentasi. lalu untuk orang awam, apa itu face demography? face liveness? contoh output gambar dari vechicle density/people density seperti apa?

Untuk itu kita bisa menambahkan 1 navigation di sidebar yang dikhususkan untuk memberi gambaran dan dokumentasi untuk setiap analytic. 

Atau lebih simplenya kita bisa memberi penjelasan dibawah setiap judul analytic 
![image](https://user-images.githubusercontent.com/45916985/133382557-92e53875-5fd7-46d9-8036-8c0400be5852.png)
(contoh doc dari open weather api)
![image](https://user-images.githubusercontent.com/45916985/133385066-e5aa53b4-8fbe-4098-8cea-0302c15c8bad.png)

Mungkin ini juga dapat mengurangi Things to note when making a request di tiap doc api yang menurut gw tampak annoying
![image](https://user-images.githubusercontent.com/45916985/133386854-db5e35ef-2386-42e7-92d9-557f1df46e0d.png)

**Comment by rizkybaihaqy** [comment]
Setuju, menambahkan penjelasan analytic juga bisa menyamakan persepsi setiap pengguna. Jujur, gw sendiri juga setiap coba satu analytic harus searching google dulu apa yang bisa gw harapin dari analytic tsb
